Clarify Ephemeral Containers RBAC (#55447)
kubernetes/website: content/en/docs/concepts/workloads/pods/ephemeral-containers.md

sharing](/docs/tasks/configure-pod-container/share-process-namespace/) so
you can view processes in other containers.

## Access control {#access-control}

Adding an ephemeral container to a running Pod uses the Pod's
`ephemeralcontainers` subresource rather than being writable via `pod.spec`.

> This is a deliberate design choice. Older admission plugins and security policies don't always recognize ephemeral containers, and treating

Reviewer: "Older" implies a timeline, while docs must be evergreen,

```suggestion
This is a deliberate design choice. Some admission plugins and security policies don't recognize ephemeral containers, and treating
```

ephemeral container creation as a distinct API surface lets cluster
administrators make an explicit decision about who may add containers to
already-running Pods.

> For the same reason, the `pods/ephemeralcontainers` subresource is *not*

Reviewer 2: (nit)

```suggestion
For the same reason, the `pods/ephemeralcontainers` subresource is **not**
```

sharing](/docs/tasks/configure-pod-container/share-process-namespace/) so
you can view processes in other containers.

## Access control {#access-control}

Adding an ephemeral container to a running Pod uses the Pod's
`ephemeralcontainers` subresource rather than being writable via `pod.spec`.
This is a deliberate design choice: admission plugins and security policies

> that predate this API may not be aware of ephemeral containers, and treating

Reviewer 3: nit: The style guide asks us to [use active voice](https://kubernetes.io/docs/contribute/style/style-guide/#use-active-voice) and avoid "may". 

```suggestion
This is a deliberate design choice. Older admission plugins and security policies don't always recognize ephemeral containers, and treating
```

ephemeral container creation as a distinct API surface lets cluster
administrators make an explicit decision about who may add containers to
already-running Pods.

> For the same reason, the `pods/ephemeralcontainers` subresource is **not**

Reviewer 3: nit (style): the [style guide](https://kubernetes.io/docs/contribute/style/style-guide/#bold) reserves **strong** for UI strings; for in-prose emphasis prefer italics. 

Consider 

```suggestion
For the same reason, the `pods/ephemeralcontainers` subresource is *not*
```
